.github/workflows/publish.yml

```diff
@@ -11,10 +11,10 @@ jobs:
     if: github.repository_owner == 'chartjs'
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
         with:
-          node-version: 12
+          node-version: current
```

Comment:
Do we actually still need to specify a version?

Reply:
Probably would be ok to just not specify a version